src/slidy/slidy.js

```diff
@@ -100,6 +100,17 @@ export function slidy (slider, options) {
     slideContainerDOMEl.style.cssText = cssText
   }
 
+  function _setTailArrowClasses () {
+    const { tailArrowClass, infinite } = options
+    if (tailArrowClass === undefined || tailArrowClass === '' || infinite) { return }
+
+    const prevArrow = frameDOMEl.getElementsByClassName(options.classNamePrevCtrl)[0]
+    prevArrow.classList[ index < 1 ? 'add' : 'remove' ](tailArrowClass)
+
+    const nextArrow = frameDOMEl.getElementsByClassName(options.classNameNextCtrl)[0]
+    nextArrow.classList[ index > options.items.length - 2 ? 'add' : 'remove' ](tailArrowClass)
```

Comment:
What does this 2 mean? I would think that nextArrow should have the tailArrowClass if index is the last item